python3-requirements.yml

```diff
@@ -81,21 +80,15 @@ modules:
       - type: file
         url: https://files.pythonhosted.org/packages/c2/f1/df59e28c642d583f7dacffb1e0965d0e00b218e0186d7858ac5233dce840/click-8.1.3-py3-none-any.whl
         sha256: bb4d8133cb15a609f44e8213d9b391b0809795062913b383c62be0ee95b1db48
-  - name: python3-typing-extensions
```

Comment:
Not required?

Reply:
It still exists (I just cleaned up the pydantic entry a bit)